.changeset/quiet-impalas-retire.md

```diff
@@ -0,0 +1,18 @@
+---
+'graphql-fixtures': major
+'@shopify/graphql-persisted': patch
+'@shopify/react-app-bridge-universal-provider': patch
+'@shopify/react-async': patch
+'@shopify/react-csrf': patch
+'@shopify/react-csrf-universal-provider': patch
+'@shopify/react-form': patch
+'@shopify/react-form-state': patch
+'@shopify/react-graphql': patch
+'@shopify/react-hydrate': patch
+'@shopify/react-i18n-universal-provider': patch
+'@shopify/react-performance': patch
+'@shopify/react-testing': patch
+'@shopify/react-universal-provider': patch
+---
+
+Upgraded @faker-js/faker from v6 to v7
```

Comment:
Changesets trigger releases of packages. Published packages do not contain test files (the `"!build/ts/**/tests/",` in the files array in the various package.json's excludes them). I think this changeset should only list packages where our changes would result in a change to the published package. If we only touch test files within a package then it can be omitted from the list.


It looks like the only package that touches files other than test files, is `graphql-fixtures`. All other packages can be removed from this list.

```suggestion
'graphql-fixtures': major
---

Upgraded `@faker-js/faker` from v6 to v7
```